Formula/crystalline.rb

```diff
@@ -0,0 +1,37 @@
+class Crystalline < Formula
+  desc "Language Server Protocol implementation for Crystal"
+  homepage "https://github.com/elbywan/crystalline"
+  url "https://github.com/elbywan/crystalline/archive/refs/tags/v0.10.0.tar.gz"
+  sha256 "26c926ba423e4b04fc52af501cd842c8255312014fc4aa1bc3735a8cd0df3426"
+  license "MIT"
+
+  livecheck do
+    url :homepage
+  end
+
```

Comment:
```suggestion
```

Reply:
Thanks for suggestion! Just to be sure: livecheck is not required anymore? Can't find more info about modern best practices around it.

Reply:
Livecheck will run using the `:git` strategy on all GitHub URL's by default. No need to specify one unless the default isn't working.